plugins/search-react/src/extensions.tsx

```diff
@@ -86,11 +86,11 @@ export type SearchResultListItemExtensionProps<Props extends {} = {}> = Props &
   >;
 
 /**
- * @internal
+ * @public
```

Comment:
I can only see this being used inside `createSearchResultListItemExtension` does it really need to be marked as public?

Reply:
Oh, that's true, I'll fix 🙏🏻

Reply:
Fixed here 🙌🏻 : https://github.com/backstage/backstage/pull/20137/commits/c444b63e33c11be54615121e4c3eb1106a95fd7c